### ISSUE TITLE
Adding a simple word+sentence copying feature for use with SSM

### ISSUE THREAD

**Issue by 1over137** [troyeguo/koodo-reader#215]
### I've read Koodo's roadmap, there is no similar feature.

- [X] Yes

### This issue only requests one feature.

- [X] Yes

### Brief Introduction

Hi,
I'm the author of [SSM](https://github.com/FreeLanguageTools/ssmtool), a tool for quickly creating Anki flashcards when reading or watching videos to learn foreign languages. 
It currently supports one-click card creation by clicking in words in browser. This feature is achieved by this [addon](https://github.com/FreeLanguageTools/ssmtool-web-companion), which injects a small content script to do so. However, it is still lacking in feature with regards to reading ebooks, like epubs, which would be very useful for language learning. 

In short, what I need is:
There should be an option to enable the following feature:
- When clicking on a word, the sentence and the word under the cursor is copied to the clipboard in the following format:
```json
{
  "sentence": "<sentence-selected>",
  "word": "<word-under-cursor>"
}
```

This will then be received by SSM for further processing before sending to Anki.

Thanks!

**Comment by troyeguo** [comment]
Thansk for the advice, but this feature might conflict with Koodo’s own selection
behavior and require a lot of code refactor. I don't think this  feature would be worth of all these trouble.

**Comment by 1over137** [comment]
> Thansk for the advice, but this feature might conflict with Koodo’s own selection
> behavior and require a lot of code refactor. I don't think this feature would be worth of all these trouble.

What's the default selection behavior? Does it select a sentence at all?
If so, it would be great if it can simply auto-copy the sentence itself

**Comment by troyeguo** [comment]
> > Thansk for the advice, but this feature might conflict with Koodo’s own selection
> > behavior and require a lot of code refactor. I don't think this feature would be worth of all these trouble.
> 
> What's the default selection behavior? Does it select a sentence at all?
> If so, it would be great if it can simply auto-copy the sentence itself

When you select a sentence, Koodo will show a pop up menu, it already contains copy option. I don't think auto-copy would be good idea for ordinary users considering most users wouldn't want software to mess up with their clipboard.
